consent-ws/src/main/java/org/broadinstitute/consent/http/cloudstore/GCSStore.java

```diff
@@ -156,6 +157,13 @@ public String putStorageDocument(InputStream stream, String type, String fileNam
         return url.toString();
     }
 
+    public Bucket getBucketMetadata() throws IOException, GeneralSecurityException {
```

Comment:
Can we get a positive/negative test for this method? I started looking in consent for this but couldn't find it, shame on us. There **are** examples in Ontology where we do this (StoreOntologyServiceTest) that uses a mocked CloudStore object which should serve as a model.